### PR TITLE
🌐[Core] 🇺🇸 Translation

### PR DESCRIPTION
**Pull request by Suchty112**

### CODE REVIEW

File: src/modules/schoolingOverview/docs/en_US.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+Especially in large associations, the list of available courses quickly becomes very long. If you take part in many training courses even then, you will quickly lose track of what's going on.
```
Comment (KBOE2) [inline src/modules/schoolingOverview/docs/en_US.md:1]: Do they also call it "course" ingane or do they use the term "schooling"?
Same thing in the translation file(s) 😉

File: src/modules/telemetry/i18n/en_US.root.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+        "not": {
+            "title": "Not supported browser",
+            "text": "Dear LSS-Manager user,<br>unfortunately the LSSM supports your current browser <b>{name}</b> not official! This means that it is not guaranteed that all functionalities will work properly. You can find a list of the browsers we support in our <a href='{wiki}/faq.html#{wikiAnchor}' class='lightbox-open'>FAQ</a>."
+        },
+        "close": "Hide hint",
+        "faqAnchor": "in-which-browsers-does-lss-manager-work"
+    },
+    "info": {
+        "close": "Do not show this note again",
+        "text": "The LSS Manager collects metadata as described in <a href='{wiki}/metadata.html' class='lightbox-open'>Wiki</a> described. During the development phase the lifting cannot be stopped!",
```
Comment (KBOE2) [inline src/modules/telemetry/i18n/en_US.root.json:16]: "lifting" does not sound right. How about changing the sentence to "During development phase, collection cannot be stopped!"?

File: src/modules/schoolingOverview/docs/en_US.md (hunk 1 above)
Comment (KBOE2) [inline src/modules/schoolingOverview/docs/en_US.md:1]: Is "courses" the term used ingame?

**Review by KBOE2 — CHANGES_REQUESTED** [review]
See comments.
Didn't have a look at all translations as that simply would be to much for now 😉
Will have a look on the travis build later to see wether the US build has succeeded as wanted.

File: src/modules/schoolingOverview/docs/en_US.md (hunk 1 above)
Comment (Suchty112) [reply src/modules/schoolingOverview/docs/en_US.md:1]: No is Courses :D https://cdn.discordapp.com/attachments/361252906806018068/769462632113242172/unknown.png

File: src/modules/telemetry/i18n/en_US.root.json (hunk 2 above)
Comment (Suchty112) [reply src/modules/telemetry/i18n/en_US.root.json]: https://github.com/LSS-Manager/LSSM-V.4/pull/149/commits/7cfdb77991098f1d83f109f7c9f47c50e97507ba fixxed with this

**Review by KBOE2 — APPROVED** [review]

**Pull request merged**